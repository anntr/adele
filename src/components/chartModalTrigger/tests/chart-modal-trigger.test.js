import React from 'react';
import ChartModalTrigger from '../chart-modal-trigger';

const category = 'company';

it('renders trigger correctly', () => {
  const wrapper = shallow(<ChartModalTrigger category={category} />, { context: { openModal: jest.fn() } });
  expect(wrapper).toMatchSnapshot();
});
