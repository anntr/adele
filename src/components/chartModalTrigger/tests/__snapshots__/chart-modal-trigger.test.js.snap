// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders trigger correctly 1`] = `
<getContext(Component)
  content={[Function]}
/>
`;
